en/concept/flight_modes.md

```diff
@@ -15,7 +15,9 @@ The sections below provide an overview of the modes, followed by a [flight mode
 
 * **Fixed wing aircraft/ rovers / boats:** 
   * **MANUAL:** The pilot's control inputs (raw user inputs from RC transmitter) are passed directly to the output mixer.
-  * **STABILIZED:** The pilot's inputs are passed as roll and pitch *angle* commands and a manual yaw command. If the RC roll and pitch sticks are centered, the autopilot regulates the roll and pitch angles to zero, hence stabilizing (leveling-out) the attitude against any wind disturbances. However, in this mode the position of the aircraft is not controlled by the autopilot, hence the position can drift due to wind.
+  * **STABILIZED:** The pilot's inputs are passed as roll and pitch angle commands to the autopilot and as yaw command directly to the output mixer to control rudder manually. If the RC roll and pitch sticks are centered, the autopilot regulates the roll and pitch angles to zero, hence stabilizing (leveling-out) the attitude against any wind disturbances. However, in this mode the position of the aircraft is not controlled by the autopilot, hence the position can drift due to wind. With nonzero roll, pitch and yaw inputs the vehicle does a coordinated turn which has zero sideslip (the acceleration in y-direction (sidewards) is zero). During a coordinated turn, the rudder is used to control the sideslip.
```

Comment:
> The pilot's inputs are passed as roll and pitch angle commands to the autopilot and as yaw command directly to the output mixer to control rudder manually.

To something like ... (the above is a little clunky, particularly " and as yaw command directly to the output mixer to control rudder manually"):

> The pilot's pitch and roll inputs are passed as angle commands to the autopilot, while the yaw input is is sent directly via the output mixer to the rudder (manual control).